doc/api/n-api.md

```diff
@@ -3277,21 +3277,20 @@ napi_status napi_get_arraybuffer_info(napi_env env,
 ```
 
 * `[in] env`: The environment that the API is invoked under.
-* `[in] arraybuffer`: `napi_value` representing the `ArrayBuffer` being queried.
+* `[in] arraybuffer`: `napi_value` representing the `ArrayBuffer` or `SharedArrayBuffer` being queried.
 * `[out] data`: The underlying data buffer of the `ArrayBuffer`. If byte\_length
```

Comment:
* `[out] data`: The underlying data buffer. ...

or if you prefer:

* `[out] data`: The underlying data buffer of the `ArrayBuffer` or  `SharedArrayBuffer` .

Reply:
thanks! I edited